packages/commonwealth/client/scripts/views/components/Comments/CommentEditor/CommentEditor.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { CWText } from '../../component_kit/cw_text';
+import { User } from 'views/components/user/user';
+import Topic from 'client/scripts/models/Topic';
+import Account from 'client/scripts/models/Account';
+import app from 'state';
+import clsx from 'clsx';
+import { ReactQuillEditor } from '../../react_quill_editor';
+import { CWButton } from '../../component_kit/new_designs/cw_button';
+import { CWValidationText } from '../../component_kit/cw_validation_text';
+import BN from 'bn.js';
+import { weiToTokens, getDecimals } from 'helpers';
+import { ContentType } from 'types';
+import type { DeltaStatic } from 'quill';
+import './CommentEditor.scss';
+
+type CommentEditorProps = {
+  parentType: ContentType;
```

Comment:
can we change `parentType` to be more explicit like and an enum `mode: 'comment' | 'reply'` or a bool `isCommenting: boolean`.

I see we only use it once like this

```
{parentType === ContentType.Comment ? 'Reply as' : 'Comment as'}
```
=> 
```
{mode === 'reply' ? 'Reply as' : 'Comment as'}
// or
{!isCommenting ? 'Reply as' : 'Comment as'}
```

Reply:
I think ContentType is being specified to make explicit that parentType may match more than a comment or a reply going forward (e.g. a proposal).